tck/methods/account.go

```diff
@@ -0,0 +1,279 @@
+package methods
+
+import (
+	"context"
+	"fmt"
+	"github.com/creachadair/jrpc2"
+	"github.com/hashgraph/hedera-sdk-go/v2"
+	"time"
+)
+
+// ---- Struct to hold hedera.Client implementation and to implement the methods of the specification ----
+type AccountService struct {
+	sdkService *SDKService
+	hedera.AccountID
+}
+
+// NOTE: Structs are created with uppercase, otherwise they are private and JSON serializer cannot encode/decode them.
+// That's why we set `json:"{expected field name}"`, which is lowercase.
+// ---- Different methods params objects ----
+type CreateAccountParams struct {
+	PublicKey                     string        `json:"publicKey"`
+	InitialBalance                int64         `json:"initialBalance"`
+	ReceiverSignatureRequired     bool          `json:"receiverSignatureRequired"`
+	MaxAutomaticTokenAssociations uint32        `json:"maxAutomaticTokenAssociations"`
+	StakedAccountId               string        `json:"stakedAccountId"`
+	StakedNodeId                  int64         `json:"stakedNodeId"`
+	DeclineStakingReward          bool          `json:"declineStakingReward"`
+	AccountMemo                   string        `json:"accountMemo"`
+	AutoRenewPeriod               time.Duration `json:"autoRenewPeriod"`
+	PrivateKey                    string        `json:"privateKey"`
+}
+type AccountFromAliasParams struct {
+	OperatorId     string `json:"operator_id"`
+	AliasAccountId string `json:"aliasAccountId"`
+	InitialBalance int64  `json:"initialBalance"`
+}
+type UpdateAccountKeyParams struct {
+	AccountId     string `json:"accountId"`
+	NewPublicKey  string `json:"newPublicKey"`
+	OldPrivateKey string `json:"oldPrivateKey"`
+	NewPrivateKey string `json:"newPrivateKey"`
+}
+type UpdateAccountMemoParams struct {
+	AccountId string `json:"accountId"`
+	Key       string `json:"key"`
+	Memo      string `json:"memo"`
+}
+type DeleteAccountParams struct {
+	AccountId   string `json:"accountId"`
+	AccountKey  string `json:"accountKey"`
+	RecipientId string `json:"recipientId"`
+}
+
+// TODO: Move this struct in dedicated file for hepler structs, or something like this. We want to use it for all methods
```

Comment:
Different file with Error definitions would be appropriate here.

Reply:
Done